engine/gamesys/src/gamesys/resources/res_sound_data.cpp

```diff
@@ -71,27 +71,28 @@ namespace dmGameSystem
         g_SoundDataContext->m_ChunkSize = chunk_size;
     }
 
-    static dmSound::SoundDataType TryToGetTypeFromBuffer(char* buffer, uint32_t bufferSize, dmSound::SoundDataType default_type)
+    static bool TryToGetTypeFromBuffer(char* buffer, uint32_t buffer_size, dmSound::SoundDataType* out)
```

Comment:
The previous code didn't make much sense, if failing, defaulting to wav, but knowing that the data is not a wav file (in my case it was a lz4 compressed ogg file, which didn't work for either decoder)